openmc/model/model.py

```diff
@@ -1015,3 +1015,53 @@ def update_material_volumes(self, names_or_ids, volume):
         """
 
         self._change_py_lib_attribs(names_or_ids, volume, 'material', 'volume')
+
+    def differentiate_burnable_mats(self, diff_volume_method: str):
```

Comment:
This is your opportunity to change it from "burnable" to "depletable"!! We don't have any issue with backward compatibility for the naming of a new method.

Reply:
awesome awesome, I didn't want to try to do to much, but seeing as you are happy for this rename then I shall add it to the PR. You have made my day twice